src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/IntegrationBaseTest.java

```diff
@@ -43,7 +45,8 @@
     protected static final int POLL_INTERVAL_MS = 100;
     protected static final int POLL_DELAY_MS = 10;
     private static final int JMS_RECEIVE_TIMEOUT = 500;
-
+    @Rule
+    public Timeout globalTimeout = Timeout.seconds(2);
```

Comment:
I think this could be achieved with the `org.junit.jupiter.api.Timeout` annotation on the class here, instead of with the old JUnit 4 rules

Reply:
I've added the `@Timeout` annotation to class level but have had to increase the timeout limit to 10 seconds. The previous rule wasn't actually being applied